pkg/config/controller_config.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"flag"
+	"fmt"
+	"github.com/spf13/pflag"
+	"k8s.io/cloud-provider/config"
+	"os"
+)
+
+const (
+	flagResourceTags     = "resource-tags"
+	flagTaggingResources = "tagging-resources"
+)
+
+var ControllerCFG = &ControllerConfig{}
+
+// ControllerConfig stores the additional flags for global usage
```

Comment:
Global configurations (and controller specific ones) tend to be added to the CloudConfig struct. Is there a reason for not doing this here?

I am also wondering about when would you add something to CloudConfig vs when would you add it as a flag.